old/calendar_generator.py

```
@@ -0,0 +1,155 @@
+            )
+        
+        # Add hotel events
+        if 'hotels' in itinerary_data:
+            for hotel in itinerary_data['hotels']:
+                add_hotel_events(cal, hotel, itinerary_data['booking_reference'])
+        
+        # Write to file
+        with open('calendar_events.ics', 'wb') as f:
+            f.write(cal.to_ical())
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)